packages/suite/src/components/wallet/TransactionItem/TransactionItem.tsx

```diff
@@ -107,6 +109,8 @@ export const TransactionItem = memo(
 
         const { descriptor: address, symbol } = useSelector(selectSelectedAccount) || {};
 
+        const networkFeatures = network.accountTypes[accountType]?.features ?? network.features;
```

Comment:
change of logic here, instead of getting a `network` with overriden account values, we look at both places.